mail/opensmtpd/Portfile

```diff
@@ -43,8 +43,10 @@ post-patch {
     reinplace "s|file:/etc/mail/aliases|file:${sysconfdir}/aliases|g" ${worksrcpath}/usr.sbin/smtpd/smtpd.conf
 }
 
-
-
+#temp fix for 7.4.0p1. Already patched upstream, to be included in next release.
+patchfiles          configureac.patch
+patchfiles          configure.patch
```

Comment:
there should be no need to patch both. It looks like the port uses the provided "configure" file, so that needs likely the patch. Only when using "autoreconf" one would need to patch the configure.ac